README.md

```diff
@@ -43,6 +43,10 @@ QUARKUS_GITHUB_APP_WEBHOOK_PROXY_URL=<your Smee.io channel URL>
 QUARKUS_GITHUB_APP_PRIVATE_KEY=-----BEGIN RSA PRIVATE KEY-----\
                  <your private key>                          \
 -----END RSA PRIVATE KEY-----
+
+QUARKUS_MAILER_FROM=<email address>
+QUARKUS_MAILER_USERNAME=<email address>
+QUARKUS_MAILER_PASSWORD=<email password>
```

Comment:
shouldn't we mention that this is optional if you don't send emails?

Reply:
Adding a note